subsys/nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -1,353 +1,1114 @@
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA2048_SHA224_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA2048_SHA256_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA2048_SHA384_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA2048_SHA512_OAEP))) \
+			|| (PSA_WANT_ALG_RSA_PSS && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA2048_SHA1_PSS)   || \
+										 (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA2048_SHA224_PSS) || \
+										 (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA2048_SHA256_PSS) || \
+										 (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA2048_SHA384_PSS) || \
+										 (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA2048_SHA512_PSS)))  \
+			|| (PSA_WANT_ALG_RSA_PKCS1V15_SIGN && ((PSA_WANT_ALG_SHA_1  && !PSA_ACCEL_RSA2048_SHA1_PKCS1V15_SIGN)     || \
+												   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA2048_SHA224_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA2048_SHA256_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA2048_SHA384_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA2048_SHA512_PKCS1V15_SIGN)))
+
+config PSA_NEED_OBERON_RSA_KEY_SIZE_3072
+	bool
+	default y
+	depends on PSA_WANT_RSA_KEY_SIZE_3072
+	depends on (PSA_WANT_ALG_RSA_PKCS1V15_CRYPT && !PSA_ACCEL_RSA3072_PKCS1V15_CRYPT) \
+			|| (PSA_WANT_ALG_RSA_OAEP && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA3072_SHA1_OAEP)     || \
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA3072_SHA224_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA3072_SHA256_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA3072_SHA384_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA3072_SHA512_OAEP))) \
+			|| (PSA_WANT_ALG_RSA_PSS && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA3072_SHA1_PSS)   || \
+										 (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA3072_SHA224_PSS) || \
+										 (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA3072_SHA256_PSS) || \
+										 (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA3072_SHA384_PSS) || \
+										 (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA3072_SHA512_PSS)))  \
+			|| (PSA_WANT_ALG_RSA_PKCS1V15_SIGN && ((PSA_WANT_ALG_SHA_1  && !PSA_ACCEL_RSA3072_SHA1_PKCS1V15_SIGN)     || \
+												   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA3072_SHA224_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA3072_SHA256_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA3072_SHA384_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA3072_SHA512_PKCS1V15_SIGN)))
+config PSA_NEED_OBERON_RSA_KEY_SIZE_4096
+	bool
+	default y
+	depends on PSA_WANT_RSA_KEY_SIZE_4096
+	depends on (PSA_WANT_ALG_RSA_PKCS1V15_CRYPT && !PSA_ACCEL_RSA4096_PKCS1V15_CRYPT) \
+			|| (PSA_WANT_ALG_RSA_OAEP && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA4096_SHA1_OAEP)     || \
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA4096_SHA224_OAEP)  || \
```

Comment:
Please align SHA_1 with the other SHA's to make it clear that SHA_1 is not a special case.

Whitespace should be used to differentiate different things, and group similar things.

Now our whitespace is communicating that there is something special about SHA_1.

The same applies elsewhere.

Reply:
I will format, the Kconfig files indeed. I will try to make them consistent.